crates/sui-framework/packages/sui-framework/sources/deny_list.move

```diff
@@ -95,7 +95,9 @@ module sui::deny_list {
         assert!(vec_set::contains(denied_addresses, &addr), ENotDenied);
         vec_set::remove(denied_addresses, &addr);
         let denied_count = table::borrow_mut(&mut list.denied_count, addr);
-        *denied_count = *denied_count - 1;
+        if (*denied_count > 0) {
```

Comment:
Wish we could plug in an invariant violation error from here

Reply:
It would be nearly identical yes (just the error code would be different)

Reply:
I would vote for assert so that an invariant violation doesn't go unnoticed.
If one day we introduce a system_invariant_assert! we can use that here.